app/.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
+  "rules": {
+    "camelcase": 0,
+    "complexity": [2, 6],
+    "consistent-return": 0,
+    "curly": [2, "all"],
+    "dot-notation": 0,
+    "eqeqeq": [2, "allow-null"],
+    "globals": {
+      "console": false,
```

Comment:
We won't need most of these globals.
Specifying an `env` of "jasmine" and/or "mocha" should give us all the "afterEach", "beforeEach", "it", "describe" functions, plus "define" should be added via the `amd` env, and "require" via `node` env. the "console" global is probably available through `browser` and `node` envs.

I think that only leaves "router".